test/unit/test_cleanup.py

```diff
@@ -0,0 +1,86 @@
+import random
+import os
+import time
```

Comment:
Sort imports alphabetically.
```suggestion
import os
import random
import time
```